WordPress/src/main/java/org/wordpress/android/ui/prefs/DotComSiteSettings.java

```diff
@@ -607,6 +609,9 @@ private JSONObject serializeDotComParamsToJSONObject() throws JSONException {
         if (mSettings.twitterUsername != null && !mSettings.twitterUsername.equals(mRemoteSettings.twitterUsername)) {
             params.put(TWITTER_USERNAME_KEY, mSettings.twitterUsername);
         }
+        if (!StringUtils.equals(mSettings.startOfWeek, mRemoteSettings.startOfWeek)) {
```

Comment:
I like the usage of a helper here, but interestingly enough, if the `mSettings.startOfWeek` is null, it'll return `false` and we'll actually try to add `null` to the `params`. I think we should do one of these things:
1. Stay consistent with the pattern used for the other settings (ex: `twitterUsername`)
2. Introduce a helper method within this class that does exactly what we want.

What do you think?